integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -400,4 +688,31 @@ private void createAuxiliaryImage(String stageDirPath, String dockerFileLocation
             .command(cmdToExecute))
         .execute(), String.format("Failed to execute", cmdToExecute));
   }
+
+  private void createSecretsForDomain(String adminSecretName, String encryptionSecretName, String domainNamespace) {
+    if (!secretExists(OCIR_SECRET_NAME, domainNamespace)) {
+      createOcirRepoSecret(domainNamespace);
+    }
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    if (!secretExists(adminSecretName, domainNamespace)) {
+      createSecretWithUsernamePassword(adminSecretName, domainNamespace,
+          ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+    }
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    if (!secretExists(encryptionSecretName, domainNamespace)) {
+      createSecretWithUsernamePassword(encryptionSecretName, domainNamespace, "weblogicenc", "weblogicenc");
+    }
+  }
+
+  private void verifyDomainEventContainsExpectedErrorMsg(OffsetDateTime timestamp, String expectedErrorMsg) {
```

Comment:
A candidate to be included in a generic utility class.

Reply:
moved to CommonTestUtils.java